src/vstest.console/Resources/Resources.resx

```diff
@@ -667,4 +667,10 @@
     <value>--DisableAutoFakes|/DisableAutoFakes:&lt;true/false&gt;
      Example: /DisableAutoFakes:true</value>
   </data>
+  <data name="CollectWithTestSettingErrorMessage" xml:space="preserve">
+    <value>--Collect|/Collect:"{0}" is not supported if test run is configured using testsettings.</value>
+  </data>
+  <data name="RunsettingsWithDCErrorMessage" xml:space="preserve">
+    <value>DataCollector with friendlyName="{0}" in runsettings in not supported if test run is configured using testsettings.</value>
```

Comment:
`Data collectors configured via run settings are not supported with embedded test settings. Please see https://aka.ms/vstest-configure-datacollector for more information. Run settings: {0}.`

The linked url should have the spec for this change and clarify that `testsettings` may have data collectors.

Reply:
Looks good. Link to https://github.com/Microsoft/vstest-docs/blob/master/docs/analyze.md.